src/stim/diagram/diagram_util.cc

```diff
@@ -38,3 +38,16 @@ size_t stim_draw_internal::utf8_char_count(const std::string &s) {
     }
     return t;
 }
+
+void stim_draw_internal::add_vector_to_ss(std::ostream &ss, std::vector<double> vec) {
```

Comment:
Give this a more descriptive name. I can't tell from the name why this would be colon first then underscore. Maybe "add_coord_summary_to_ss", for example